pkg/prometheus/promcfg_test.go

```diff
@@ -50,6 +50,120 @@ func TestConfigGeneration(t *testing.T) {
 	}
 }
 
+func TestGlobalSettings(t *testing.T) {
```

Comment:
What is the GlobalSettings test?

Reply:
It is testing the prometheus parameters described below and was introduced as part of testing a previous way of setting honor_timestamps. Now I can move it to other PR or leave it here:
```
evaluation_interval
scrape_interval
external_labels
```